src/config/dynamic-theme-fixes.config

```diff
@@ -18671,6 +18675,14 @@ html[data-darkreader-scheme="dark"] img[src*="Eagle_with_fasces.svg"] {
     background-color: ${#BBBBBB} !important;
     mix-blend-mode: screen !important;
 }
+body.mediawiki.skin-monobook:has(> #globalWrapper) {
+    background-color: #f9f9f9 !important;
+    background-image: linear-gradient(#0000, #0000), url(https://upload.wikimedia.org/wikipedia/commons/f/fb/Monobook_skin_background.svg) !important;
+    filter: invert();
+}
+body.mediawiki.skin-monobook > * {
```

Comment:
As we remove the invert from previous selector we need to make some changes here so
We change this selector to:
```
html[data-darkreader-scheme="dark"] body.mediawiki.skin-monobook > *,
html[data-darkreader-scheme="dark"] body.mediawiki.skin-monobook:has(> #globalWrapper) {
```

This will apply invert only when dark scheme is applied.